core/runtime/src/main/java/io/quarkus/runtime/ApplicationLifecycleManager.java

```diff
@@ -27,7 +27,7 @@
  * The {@link Application} class is responsible for starting and stopping the application,
  * but nothing else. This class can be used to run both persistent applications that will run
  * till they receive a signal, and command mode applications that will run until the main method
- * returns. This class registers a shutdown hook to properly shut down the application, and handles
+ * returns. This class registers a shutªdown hook to properly shut down the application, and handles
```

Comment:
WTF happened here? ;)

Reply:
oops.  i switch between windows and mac regularly and the muscle sometimes lags.  :)